src/main/java/se/sundsvall/byggrintegrator/integration/byggr/ByggrIntegrationMapper.java

```diff
@@ -100,11 +101,21 @@ private List<Arende> extractErrands(GetRelateradeArendenByPersOrgNrAndRoleRespon
 	private ByggrErrandDto toByggErrandDto(Arende arende) {
 		return ByggrErrandDto.builder()
 			.withByggrCaseNumber(arende.getDnr())
+			.withDescription(arende.getBeskrivning())
+			.withPropertyDesignation(toPropertyDesignation(arende))
 			.withEvents(toEvents(arende.getHandelseLista()))
 			.withStakeholders(toStakeholders(arende.getIntressentLista()))
 			.build();
 	}
 
+	private String toPropertyDesignation(Arende arende) {
+		return ofNullable(arende)
+			.map(a -> (ArendeFastighet) a.getObjektLista().getAbstractArendeObjekt().getFirst())
```

Comment:
Can there be objects of other class than ArrendeFastighet here? In that case you might end up with a class cast exception, right? Change to:
```
		return ofNullable(arende)
			.map(a -> a.getObjektLista().getAbstractArendeObjekt())
			.filter(CollectionUtils::isNotEmpty)
			.map(aObjects -> aObjects.getFirst())
			.filter(aObject -> ArendeFastighet.class.isInstance(aObject))
			.map(ArendeFastighet.class::cast)
			.map(ArendeFastighet::getFastighet)
			.filter(Objects::nonNull)
			.map(f -> f.getTrakt() + " " + f.getFbetNr())
			.orElse(null);
```
to avoid CCE (and possible NPE on empty list and fastighet eq null)?